pages/early-access/index.html

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Get early access to Patcher
+permalink: /early-access/
+slug: early-access
+custom_js:
+- contact
```

Comment:
Check if we can use the same script from the contact, or if we need another one

Reply:
All forms use the same one, should be fine.